src/process/tests.rs

```diff
@@ -221,7 +221,6 @@ async fn no_description_none_required_pr_actions() {
 async fn review_not_required_if_label_not_configured() {
     use ReviewState::{Approved, ChangesRequested, Commented};
 
-    #[track_caller]
```

Comment:
nit: I think should keep this as possible test failures will point at the corresponding `assert_approved` below, instead of where this function calls `assert_eq`

Reply:
I can re-add it, but according to https://github.com/rust-lang/rust/issues/87417 it will do nothing (if I understand it correctly). I guess it will eventually be fixed, so I can re-add and ignore the warning instead

Reply:
updated